jobs/promptLicenceCreation.ts

```diff
@@ -1,6 +1,6 @@
 import 'reflect-metadata'
 import _ from 'lodash'
-import { add, startOfWeek, endOfWeek } from 'date-fns'
+import { add, subDays, startOfISOWeek, endOfISOWeek } from 'date-fns'
```

Comment:
replacing startOfWeek with startOfISOWeek as week will automatically start on a Monday.
same for endOfISOWeek